src/main/java/com/sap/oss/phosphor/fosstars/model/value/SemanticVersion.java

```diff
@@ -4,6 +4,7 @@
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
Was it intended to use `SpringUtils` from the Plexus/Codehaus library?
I would recommend to not add a dependency to such a library because of an `StringUtils.isEmpty(..)` check.

Reply:
It was intended. The version check was just for null. I wanted to check for the `""` empty string as well. If you feel that we do not need this check. And let the parse matcher take over. I do not mind that also. I will revert it back to the original.

Reply:
I did not add any new dependency though. It was part of something else.

Reply:
But now that I think about this. We do not need an empty string check. The parse matcher already does that for us.

Reply:
Reason why I asked was that I recommend to only use classes of direct dependencies and not from indirect dependencies 😉
And to not add a new dependency only because of something like a null/empty check for a string.

Hope this way my comment is more clear 🙂
However, removing the check when it is not required is the best solution 😉

```
+- org.apache.maven:maven-model:jar:3.8.1:compile
|  \- org.codehaus.plexus:plexus-utils:jar:3.2.1:compile
```

Reply:
Yes, please use `StringUtils` from Apache Commons.